analysis_options.yaml

```diff
@@ -16,20 +16,30 @@ linter:
     - annotate_overrides
     - avoid_annotating_with_dynamic
 #    - avoid_as
+#    - avoid_bool_literals_in_conditional_expressions # not yet tested
```

Comment:
Actually, on reading this again, I'm not sure this note is totally accurate.  Maybe a link to #1068 instead?  Maybe something like:

    # under review (see #1068)